### DISCUSSION TITLE
index code에 대한 질문

### DISCUSSION THREAD

**Discussion by Youkwangjin** [Hyorim-Kim/Team#55] [Q&A] [answered]
index.html 보여지는 화면에 비해서 코드가 간결하던데 어떤식으로 구성했나요?

**Comment by Hyorim-Kim** [comment, marked as answer]
다른 페이지와 중복되는 코드 부분은 새로운 템플릿 파일로 분리한 후 삽입하여 코드의 길이를 줄였습니다.
